wvd-sh/WVD scaling script/Config-MSI.Json

```diff
@@ -29,22 +29,27 @@
 				},
 				{
 					"_Comments": "Hostpool session Load Balancing Type",
-					"Name": "PeakLoadBalancingType",
+					"Name": "peakLoadBalancingType",
 					"Value": "BreadthFirst"
 				},
 				{
 					"_Comments": "Session Load Balancing Peak Hours",
 					"Name": "sessionLoadBalancingPeakHours",
 					"Value": "9:00-3:00"
 				},
+				{
+					"_Comments": "Name of the Maintainance Tag",
+					"Name": "maintainanceTagName",
```

Comment:
Replace "maintananceTagName" with "maintanenceTagName" everywhere.

Reply:
Thank you Christian to addressing us. We have replaced in both the scripts.